src/types.ts

```diff
@@ -331,6 +331,13 @@ export interface PostHogConfig {
      */
     rageclick: boolean
 
+    /**
+     * Determines whether PostHog should automatically capture navigation events using the History API and emit them as pageviews.
+     *
+     * @default false
+     */
+    capture_history_events: boolean
```

Comment:
How about adding another option to `capture_pageviews`, e.g. `capture_pageviews: 'history-change`?

This is what people might expect `capture_pageviews` to do anyway

Reply:
Done :)